Documentation/contributing/testing/unit.rst

```diff
@@ -63,6 +63,31 @@ run the unit tests and tear down the prerequisites:
 
     $ make unit-tests TESTPKGS=github.com/cilium/cilium/pkg/kvstore
 
+There are some 'privileged' tests in some packages. Those privileged tests are
+not run by default when you run the unit tests for the respective package.
+The privileged test files have an entry at the top of the test file as shown.
+
+::
+
+    +build privileged_tests
+
+There are two ways that you can run the 'privileged' tests. All privileged tests
+have to be run as 'root' user.
```

Comment:
I would move the "all tests have to be run as root" to the previous paragraph, as that would flow nicer.

Reply:
I'd actually remove that sentence. I think it's clear from context and commands below that tests should be run as root.
```suggestion
There are two ways that you can run the 'privileged' tests.
```

Reply:
Done